skaffold.yaml

```diff
@@ -46,6 +46,13 @@ deploy:
           otel.logs.filter.exclude.record_attributes[0].value: "monitoring"
           cluster.name: cluster name
           cluster.uid: cluster-uid-123456789
+          # excluded annotations:
+          # * kubectl.kubernetes.io/last-applied-configuration - deployed by kubectl, contains full config for each resource
+          # * cni.projectcalico.org/ - deployed in CI system (k3s)
+          otel.metrics.k8s_instrumentation.annotations.excludePattern: "(kubectl\\.kubernetes\\.io/last-applied-configuration)|(cni\\.projectcalico\\.org/.*)"
```

Comment:
Does it make sense to exclude these also when deployed to an actual cluster?

Reply:
I am not sure. @jaroslav-fedor-swi do you want us to collect all annotations, including `last-applied-configuration` that is set by `kubectl apply` and includes json of full object?